### ISSUE TITLE
Add lerpRotateTo functionality

### ISSUE THREAD

**Issue by mcfly3001** [yomotsu/camera-controls#131]
The rotateTo method provides the transition flag for allowing a smooth transition to a given polar orientation. In my use case though I would like to be able to provide a deltaT to this method to be able to set intermediate orientations. My plan is to use the wheel event to incremental raise the deltaT until the target viewing direction is reached.
Therefore a lerpRotateTo method would be ideal. Any feeback or tips if there is another obvious solution would be welcome.

**Comment by yomotsu** [comment]
Thanks for your suggestion.
How about just using `THREE.MathUtils.lerp`?
```js
const azimuthAngle = THREE.MathUtils.lerp( azimuthStart, azimuthEnd, t );
const polarAngle   = THREE.MathUtils.lerp( polarStart,   polarEnd,   t );
cameraControls.rotateTo( azimuthAngle, polarAngle, true );
```

**Comment by mcfly3001** [comment]
Hi,
works at least much better than my current approach, thanks! First thing I tried was to change the target view direction on each deltaT update. I recalculated polar and azimuth each time with the atan2 function which lead to weird rotations at border cases (e.g. the azimuth flipped from -2.xxx to +3.xxx).
With your suggestion it seems to work better, though sometimes the camera does not use the shortest angle when rotating to the new direction. But I hope to figure this one out on my own.
Thanks!
